src/views/product-downloads-view/components/release-information/release-information.module.css

```diff
@@ -4,13 +4,16 @@
  */
 
 .root {
+	margin-top: 16px;
 	margin-bottom: 24px;
 }
 
+.headingContainer {
+	margin: 56px 0 16px 0;
```

Comment:
everyday our magic numbers hurt me a bit more inside. I might see if I have some time at year end to make a draft PR of us moving away from them.